buildSrc/src/main/groovy/org/apache/geode/gradle/plugins/DependencyConstraints.groovy

```diff
@@ -272,5 +272,10 @@ class DependencyConstraints implements Plugin<Project> {
     dependencySet(group: 'org.springframework.session', version: '2.4.1') {
       entry('spring-session-data-redis')
     }
+
```

Comment:
Are we expecting to have more modules from `org.jetbrains` in the product? If not, then the syntax above for individual modules is a little cleaner.

Reply:
@pivotal-jbarrett  asked explicitly for this syntax. I don't have an opinion on it

Reply:
Flip a coin...